### ISSUE TITLE
[1.17.1] - Mutation failing silently when initiated from a player's client.

### ISSUE THREAD

**Issue by thatlonelybugbear** [trioderegion/warpgate#136]
Tested in Foundry v10.291 and v11.301 with only Warpgate enabled. Dnd5e 2.2.1

The request never appears on GM's account and `Always accepting mutation requests` makes no difference.

Mutation line triggered from console: 
```js
makeToasts = await warpgate.mutate(game.user.targets.first().document,{actor:{"system.attributes.hp.value":15}},{},{})
```
```js
Token Delta: {}
Actor Delta: {
    "system": {
        "attributes": {
            "hp": {
                "value": 256
            }
        }
    }
}
Embedded Delta:{}
```
```js
Received socket data => {
    "op": "req-mutate",
    "payload": {
        "userId": "Ie3Pb0obTnSDZ9xL",
        "tokenId": "B6h4EdOxIcM64gIL",
        "sceneId": "GDn5FWPqQtbVK8IP",
        "updates": {
            "actor": {
                "system": {
                    "attributes": {
                        "hp": {
                            "value": 15
                        }
                    }
                },
                "flags": {
                    "warpgate": {
                        "mutate": [
                            {
                                "delta": {
                                    "actor": {
                                        "system": {
                                            "attributes": {
                                                "hp": {
                                                    "value": 256
                                                }
                                            }
                                        }
                                    }
                                },
                                "user": "Ie3Pb0obTnSDZ9xL",
                                "comparisonKeys": {
                                    "ActiveEffect": "label"
                                },
                                "name": "mzKX4goDwvvNwIVK",
                                "updateOpts": {},
                                "overrides": {
                                    "preserveData": true
                                }
                            }
                        ]
                    }
                }
            }
        },
        "options": {
            "overrides": {
                "preserveData": true
            },
            "comparisonKeys": {
                "ActiveEffect": "label"
            },
            "updateOpts": {},
            "name": "mzKX4goDwvvNwIVK",
            "delta": {
                "actor": {
                    "system": {
                        "attributes": {
                            "hp": {
                                "value": 256
                            }
                        }
                    }
                }
            }
        }
    }
}
```
```js
Warp Gate  |  Routing operation:  req-mutate

makeToasts
VM24318:1 Uncaught ReferenceError: makeToasts is not defined
    at <anonymous>:1:1
```

![image](https://github.com/trioderegion/warpgate/assets/7237090/fe32cbf3-b49e-4d3f-87f2-ed843ff938f0)

**Comment by trioderegion** [comment]
Was this addressed in 1.17.2?

**Comment by thatlonelybugbear** [comment]
Indeed 👍
